test/functional/api/v2/media_controller_test.rb

```diff
@@ -26,7 +26,7 @@ class Api::V2::MediaControllerTest < ActionController::TestCase
     assert_difference('Medium.count', +1) do
       post :create, { :medium => new_medium }
     end
-    assert_response :created
+    assert_response :success
```

Comment:
Ah you're right, I thought it stayed true after save - so do check `request.post?` then.  It looks like the other API POST routes I was concerned about don't actually use these methods, but please just double check.